FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -164,6 +164,22 @@ final class ComposeViewController: TableNodeViewController {
         self.contextToSend.recipients = [ComposeMessageRecipient(email: "tom@flowcrypt.com", state: decorator.recipientIdleState)]
     }
 
+    private func observeComposeUpdates() {
+        composeMessageService.$state
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] state in
+                guard let message = state.message else {
+                    self?.hideSpinner()
+                    return
+                }
+                self?.updateSpinner(
+                    label: message,
+                    progress: state.progress,
+                    systemImageName: "checkmark.circle"
+                )
+            }
+            .store(in: &lifeTimeCancellable)
+    }
```

Comment:
Instead of an observable, could you please use a regular callback? A bit closer to how it was before. I find it simpler and easier to work with.

Reply:
Use same approach as for search functionality and photosManager inside same controller. 
ok, no problems. will rework to use callbacks instead